.gitignore

```diff
@@ -8,4 +8,5 @@ __pycache__
 !templates/**/lib
 !templates/**/include
 test
-*.pyc
\ No newline at end of file
+*.pyc
+knightos.sh
```

Comment:
The runner script. Didn't want to overwrite the existing knightos script. Previously that script was just installed directly to the destination.